src/PopEngine/index.ts

```diff
@@ -260,7 +261,11 @@ export class PopEngine {
 
     if (isAlreadyShowing && !!container && !!pop.opts.reusePopover) {
       // if pop is already showing for group, reuse
-      container.removeChild(container.getElementsByClassName('pop-content')[0]);
+      let contentEl : Element = <Element>container.getElementsByClassName('pop-content')[0];
+      // race condition guard
+      if (contentEl) {
+        container.removeChild(contentEl);
```

Comment:
I see. So this prevents us from potentially removing a child that doesn't exist within the popover container anymore/a child that never was added to the container? Smart!! 👍